<!DOCTYPE HTML>
<!--
	Spatial by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>

<head>
	<title>Soroor Hediyeh-zadeh</title>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<meta name="description" content="" />
	<meta name="keywords" content="" />
	<noscript>
		<link rel="stylesheet" href="css/style.css" />
		<link rel="stylesheet" href="css/skel.css" />
		<link rel="stylesheet" href="css/style-xlarge.css" />
	</noscript>
	<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Raleway:400,700">
	<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
	<link rel="stylesheet" href="css/font-awesome.min.css">
	<!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
	<script src="js/jquery.min.js"></script>
	<script src="js/skel.min.js"></script>
	<script src="js/skel-layers.min.js"></script>
	<script src="js/init.js"></script>
	<script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>
</head>

<body class="landing">

	<!-- Header -->
	<header id="header">
		<ul class="icons">
			<li>
				<a href="https://scholar.google.com/citations?user=5rg6ljIAAAAJ&hl=en" class="icon fa-graduation-cap"></a>
			</li>
			<li>
				<a href="https://github.com/soroorh" class="icon fa-github"></a>
			</li>
			<li>
				<a href="https://twitter.com/soroorh1" class="icon fa-twitter"></a>
			</li>
			<li>
				<a href="https://de.linkedin.com/in/soroor-hediyeh-zadeh-958086278" class="icon fa-linkedin"></a>
			</li>
		</ul>
		<nav id="nav">
			<ul>
				<li><a href="/" class="active">Home</a></li>
				<li><a href="/#two">Publications</a></li>
				<!-- <li><a href="/graph-convolutional-networks/">Blog</a></li> -->
				<li><a href="/#footer">Contact</a></li>
			</ul>
		</nav>
	</header>

	<!-- One -->
	<section id="one" class="wrapper style1">
		<div class="container 80%">
			<div class="row 200%">
				<div class="4u 12u$(medium)">
<!-- 					<header class="major"> -->
					<h2></h2>
					<p>
						<div class="image rounded" style="margin-bottom: 1.5em;"><img src="images/githubpages.png" width="200" alt="" /></div>
					</p>
					<h3>Soroor Hediyeh-zadeh</h3>
					<p></p>
<!-- 					</header> -->
				</div>
				<div class="8u$ 12u$(medium)">
					<p>I am a Graduate Research Student in the Department of Medical Biology (WEHI) at The University of Melbourne. Previously, I was a Research Assistant (RA) in <a href="https://www.wehi.edu.au/people/melissa-davis" target="_blank">A/Prof. Melissa Davis's Lab</a> at the Bioinformatics Division in WEHI, and a Masters student at The University of Melbourne in the School of Mathematics and Statistics. I am broadly interested in developing and studying machine learning and statistical models for the analysis of transcriptomics and proteomics data. </p>


					<p>During my RA position, I worked on machine learning models for detection of rare immune and phenotype-specific gene expression signatures in highly heterogenous patient cancer samples. I also worked on applications of Regularized Generalized Linear Models to biomedical data. In my MPhil, I frequently use and develop techniques in Representation Learning.</p>
				</div>
			</div>
		</div>

		<div class="container 75%">
			<h3 style="text-align: left">News</h3>
			<ul>
				<li>11/2023: Our work on <a href='https://www.biorxiv.org/content/biorxiv/early/2023/04/04/2023.04.02.535231.full.pdf'>modeling quantification uncertainty</a> in differential transcript expression was accepted in NAR!</li>
				<li>10/2023: Our work <a href='https://www.nature.com/articles/s41592-023-02035-2'>scPoli</a> is out in Nature Methods. scPoli also recieved the Helmholtz Munich Best Paper award!
				<li>10/2023: New Preprint Alert! Checkout <a href='https://www.biorxiv.org/content/10.1101/2023.09.08.556842.full.pdf'>scDECAF</a>, learning vector representation of gene sets for fast query of gene programs in single cell transcriptomic data!</li> 
				<li>04/2023: msImpute is out in <a href='https://www.mcponline.org/article/S1535-9476(23)00068-3/fulltext'>Molecular and Cellular Proteomics</a>. Checkout latest <a href='https://github.com/DavisLaboratory/msImpute-reproducibility/tree/main/figures'>tutorials</a> here. Find the software on <a href='https://bioconductor.org/packages/release/bioc/html/msImpute.html'>Bioconductor</a>.</li>
				<li>03/2023: Watch the recording of my talk on ContinualAI seminar series <a href='https://www.youtube.com/watch?v=T2IYBSyug6w'>here</a>.</li> 
				<li>11/2022: Officially graduated from The University of Melbourne with a MPhil degree in Computational Biology. In my thesis, I developed machine- and deep learning algorithms for (label-free) mass spectrometry proteomics.</li>
				<li>05/2022: Our workshop <a href='https://icml-compbio.github.io/icml-website-2022/2022/papers/WCBICML2022_paper_68.pdf'> paper</a> on Continual Learning for single cell biology accepted at 2022 ICML Workshop in Computational Biology! </li>
				<li>9/2021: I moved to Helmholtz Munich to do a PhD!</li>
				<li>08/2021: I presented msImpute.v2 at BioC 2021. Find the poster <a href='https://doi.org/10.7490/f1000research.1118713.1'> here</a>. Update - msImpute.v2 got an Invited Talk in <a href='https://bioconductor.github.io/BiocAsia2021/'> BioC Asia 2021</a>. Stay tuned for the details! </li>
				<li>06/2021: Our works on <a href='https://github.com/DavisLaboratory/scDECAF'>learning geneset representations in single cell RNAseq</a> and deep representation learning of TIMS-MS1 features in mass spec are accepted at ICML 2021 Computational Biology Workshop. Come and see us at poster session!</li>
				<li>04/2021: Our original work on using Machine Learning to identify ILC2 infiltration in cancer samples is now accepted in Nature Immunology. A great collaboration with Belz Lab at UQ Diamantina and WEHI. [<a href='https://www.nature.com/articles/s41590-021-00943-z'>Link</a>]</li>
				<li>03/2021: Our work with Wainwright Lab in UQ Diamantina Institute on RNA-seq and SWATH/MS proteomics analysis of Elp2 mutation in neural development is accepted in Nature Communications. [<a href='https://www.nature.com/articles/s41467-021-22888-5'>Link</a>] </li>
				<li>02/2021: Our work on prediction of ILC2 infiltration in Colorectal Cancer is published in Cancers. Joint work with Belz Lab at WEHI. [<a href="https://www.mdpi.com/2072-6694/13/3/559/htm">Link</a>] </li>
				<!-- <li>09/2020: Our paper on <a href="https://arxiv.org/abs/2006.15055">Object-centric Learning with Slot Attention</a> is accepted for spotlight presentation at NeurIPS!</li>
				<li>07/2020: I gave two invited workshop talks at ICML: <a href="https://slideslive.com/38930703/attentive-grouping-and-gnns-for-objectcentric-learning">Attentive Grouping</a> and <a href="https://slideslive.com/38930558/relational-structure-discovery">Relational Structure Discovery</a>.</li>
				<li>04/2020: I have defended my PhD thesis with distinction "cum laude" (awarded 3x in the past 10 years at our institute). My thesis on "Deep Learning with Graph-Structured Representations" is available <a href="https://hdl.handle.net/11245.1/1b63b965-24c4-4bcd-aabb-b849056fa76d">here</a>.</li>
				<li>01/2020: I have joined Google Brain as a Research Scientist in Amsterdam.</li>
				<li>12/2019: I am co-organizing the <a href="https://geometric-relational-dl.github.io">ELLIS Workshop on Geometric and Relational Deep Learning</a>.</li>
				<li>12/2019: Our work on <a href="https://arxiv.org/abs/1812.01483">Contrastive Learning of Structured World Models</a> is accepted at <a href="https://iclr.cc">ICLR 2020</a> as an oral presentation.</li>
				<li>08/2019: I am co-organizing the <a href="https://grlearning.github.io">Graph Representation Learning</a> workshop at NeurIPS 2019.</li>
				<li>05/2019: I gave a tutorial on Unsupervised Learning with Graph Neural Networks at the <a href="http://www.ipam.ucla.edu/programs/workshops/workshop-iv-deep-geometric-learning-of-big-data-and-applications/?tab=overview">UCLA IPAM Workshop on Deep Geometric Learning of Big Data</a> (<a href="http://helper.ipam.ucla.edu/publications/glws4/glws4_15546.pdf">slides</a>, <a href="http://www.ipam.ucla.edu/programs/workshops/workshop-iv-deep-geometric-learning-of-big-data-and-applications/?tab=schedule">video</a>).</li>
				<li>04/2019: Our work on <a href="https://arxiv.org/abs/1812.01483">Compositional Imitation Learning</a> is accepted at <a href="https://icml.cc">ICML 2019</a> as a long oral.</li>
				<li>03/2019: I am co-organizing two workshops: <a href="https://rlgm.github.io">Representation Learning on Graphs and Manifolds</a> (ICLR 2019) and <a href="https://graphreason.github.io">Learning and Reasoning with Graph-Structured Data</a> (ICML 2019).</li> -->
				<!-- <li>12/2018: My DeepMind internship project <a href="https://arxiv.org/abs/1812.01483">Compositional Imitation Learning: Explaining and executing one task at a time</a> is accepted as a contributed talk at the Learning by Instruction Workshop at NeurIPS 2018.</li> -->
				<!-- <li>06/2018: One accepted paper at <a href="http://www.auai.org/uai2018/">UAI 2018</a>: <a href="https://arxiv.org/abs/1804.00891">Hyperspherical Variational Auto-Encoders</a> (plenary talk), a short paper at <a href="https://midl.amsterdam">MIDL 2018</a> on <a href="https://arxiv.org/abs/1804.04436">Extraction of Airways using Graph Neural Networks</a> and a spotlight paper at the <a href="http://www.kdd.org/kdd2018/deep-learning-day">KDD Deep Learning Day 2018</a> on <a href="https://arxiv.org/abs/1706.02263">Graph Convolutional Matrix Completion</a></li> -->
				<!-- <li>06/2018: Our work on <a href="https://arxiv.org/abs/1703.06103">Relational GCNs</a> has won the best student research paper award at ESWC 2018.</li> -->
				<!-- <li>05/2018: Our work on learning latent interaction graphs: <a href="https://arxiv.org/abs/1802.04687">Neural Relational Inference for Interacting Systems</a> is accepted as a conference paper at <a href="https://icml.cc">ICML 2018</a>.</li> -->
				<!-- <li>03/2018: I will be joining Google DeepMind for a summer internship in 2018.</li>
				<li>03/2018: Our paper <a href="https://arxiv.org/abs/1703.06103">Modeling Relational Data with Graph Convolutional Networks</a> is accepted as a conference paper at <a href="https://2018.eswc-conferences.org">ESWC 2018</a>.</li> -->
				<!-- <li>2017: In the summer of 2017, I interned at Apple in <a href="https://homes.cs.washington.edu/~guestrin/index.html">Carlos Guestrin</a>'s Machine Learning team. I worked on self-attention models for sequence tagging and classification.</li> -->
			</ul>
		</div>
	</section>

	<!-- Two -->
	<section id="two" class="wrapper style2">
		<div class="container 75%">
			<h2 style="text-align: center">Research and Publications</h2>
			<hr />
<!-- 			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
					<div class="image rounded"><img src="images/cancers2021.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Type 2 Innate Lymphoid Cells Protect against Colorectal Cancer Progression and Predict Improved Patient Survival</h4>
						<p>Identification of ILC2 populations in Colorectal Cancer by Machine Learning</p>
					</header>
					<p>Huang, Q., Jacquelot, N., Preaudet, A.,<span style="text-decoration: underline;"> Hediyeh-Zadeh, S.</span>, Souza-Fonseca-Guimaraes, F., McKenzie, A. N., ... & Belz, G. T. (2021). <span style="font-weight: bold">Type 2 Innate Lymphoid Cells Protect against Colorectal Cancer Progression and Predict Improved Patient Survival</span>. Cancers [<a href='https://www.mdpi.com/2072-6694/13/3/559/htm'>Link</a>]</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
					<div class="image rounded"><img src="images/advances.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Cotargeting BCL-2 and MCL-1 in high-risk B-ALL</h4>
						<p>Logistic regression with group lasso penalty were employed to identify gene families contributing to resistant phenotype in B-ALL</p>
					</header>
					<p>Moujalled, D. M., Hanna, D. T.,<span style="text-decoration: underline;">Hediyeh-Zadeh, S.</span>,  Pomilio, G., Brown, L., Litalien, V., ... & Khaw, S. L. (2020).<span style="font-weight: bold">Cotargeting BCL-2 and MCL-1 in high-risk B-ALL</span>. Blood advances[<a href="https://ashpublications.org/bloodadvances/article/4/12/2762/461057">Link</a>]</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/erg_natcomms.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>An Erg-driven transcriptional program controls B cell lymphopoiesis</h4>
						<p>Multi-omics data analysis, Gene Regulation Network construction </p>
					</header>
					<p>Ng, A. P., Coughlan, H. D.,<span style="text-decoration: underline;"> Hediyeh-Zadeh, S.</span>, Behrens, K., Johanson, T. M., Low, M. S. Y., ... & Alexander, W. S. (2020).<span style="font-weight: bold">An Erg-driven transcriptional program controls B cell lymphopoiesis</span>. Nature Communications [<a href='https://www.nature.com/articles/s41467-020-16828-y'>Link</a>]</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/jem_20181778_fig5.jpeg" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>TCF-1 limits the formation of Tc17 cells via repression of the MAF–RORγt axis </h4>
						<p>Identification of rare immune phenotypes in cancer samples using cell-line mouse models and Machine Learning</p>
					</header>
					<p>Mielke, L. A., Liao, Y., ..., <span style="text-decoration: underline;">Hediyeh-Zadeh, S.</span> [11/24],...& Belz, G. T. (2019).<span style="font-weight: bold">TCF-1 limits the formation of Tc17 cells via repression of the MAF–RORγt axis</span>. JEM [<a href='https://rupress.org/jem/article/216/7/1682/120672'>Link</a>]</p>
				</div>
			</div>
			<hr /> -->
			<!-- <div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/nri.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Neural Relational Inference for Interacting Systems</h4>
						<p>Learning the latent interaction graph of a dynamical system</p>
					</header>
					<p><span style="text-decoration: underline;">T. Kipf*</span>, E. Fetaya*, K. Wang, M. Welling, R. Zemel, <span style="font-weight: bold">Neural Relational Inference for Interacting Systems</span>, (ICML 2018) [<a href='https://arxiv.org/abs/1802.04687'>Link</a>, <a
										href="https://arxiv.org/pdf/1802.04687.pdf">PDF (arXiv)</a>, <a href="https://github.com/ethanfetaya/nri">code</a>], *equal contribution.</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/molgan.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>MolGAN: An implicit generative model for small molecular graphs</h4>
						<p>Learning to generate molecular graphs using a combined GAN/RL-based objective</p>
					</header>
					<p>N. De Cao, <span style="text-decoration: underline;">T. Kipf</span>, <span style="font-weight: bold">MolGAN: An implicit generative model for small molecular graphs</span>, ICML Deep Generative Models Workshop (2018) [<a href='https://arxiv.org/abs/1805.11973'>Link</a>, <a
										href="https://arxiv.org/pdf/1805.11973.pdf">PDF (arXiv)</a>, <a href="https://github.com/nicola-decao/MolGAN">code</a>].</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/svae.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Hyperspherical Variational Auto-Encoders</h4>
						<p>Learning hyperspherical latent spaces</p>
					</header>
					<p>T. R. Davidson*, L. Falorsi*, N. De Cao*, <span style="text-decoration: underline;">T. Kipf</span>, J. M. Tomczak, <span style="font-weight: bold">Hyperspherical Variational Auto-Encoders</span>, (UAI 2018), <span style="font-weight: bold; color: #DC143C">Plenary Talk</span> [<a href='https://arxiv.org/abs/1804.00891'>Link</a>, <a
										href="https://arxiv.org/pdf/1804.00891.pdf">PDF (arXiv)</a>, <a href="http://github.com/nicola-decao/s-vae">code</a>, <a href="https://nicola-decao.github.io/s-vae">blog</a>], *equal contribution.</p>
				</div>
			</div>
			<hr /> -->
			<!-- <div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
						<div class="image rounded"><img src="images/airway.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Extraction of Airways using Graph Neural Networks</h4>
						<p>Graph refinement with graph neural nets</p>
					</header>
					<p>R. Selvan, <span style="text-decoration: underline;">T. Kipf</span>, M. Welling, J. H. Pedersen, J. Petersen, M. de Bruijne, <span style="font-weight: bold">Extraction of Airways using Graph Neural Networks</span>, (MIDL 2018 - short paper) [<a href='https://arxiv.org/abs/1804.04436'>Link</a>, <a
										href="https://arxiv.org/pdf/1804.04436.pdf">PDF (arXiv)</a>].</p>
				</div>
			</div>
			<hr /> -->
			<!-- <div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
					<div class="image rounded"><img src="images/rgcn.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Modeling Relational Data with Graph Convolutional Networks</h4>
						<p>Link prediction and entity classification on knowledge graphs</p>
					</header>
					<p>M. Schlichtkrull*, <span style="text-decoration: underline;">T. N. Kipf*</span>, P. Bloem, R. vd Berg, I. Titov, M. Welling, <span style="font-weight: bold">Modeling Relational Data with Graph Convolutional Networks</span>, (ESWC 2018), <span style="font-weight: bold; color: #DC143C">Best Student Research Paper</span> [<a href='https://arxiv.org/abs/1703.06103'>Link</a>, <a
										href="https://arxiv.org/pdf/1703.06103.pdf">PDF (arXiv)</a>, <a href="https://github.com/tkipf/relational-gcn">code</a>], *equal contribution.</p>
				</div>
			</div>
			<hr /> -->
			<!-- <div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
					<div class="image rounded"><img src="images/gcmc.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Graph Convolutional Matrix Completion</h4>
						<p>Collaborative filtering with side information in bipartite graphs</p>
					</header>
					<p>R. vd Berg, <span style="text-decoration: underline;">T. N. Kipf</span>, M. Welling, <span style="font-weight: bold">Graph Convolutional Matrix Completion</span>, KDD Deep Learning Day (2018), <span style="font-weight: bold; color: #DC143C">Spotlight Talk</span> [<a href='https://arxiv.org/abs/1706.02263'>Link</a>, <a
										href="https://arxiv.org/pdf/1706.02263.pdf">PDF (arXiv)</a>, <a href="https://github.com/riannevdberg/gc-mc">code</a>]</p>
				</div>
			</div>
			<hr /> -->
			<!-- <div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
					<div class="image rounded"><img src="images/gcn.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Semi-Supervised Classification with Graph Convolutional Networks</h4>
						<p>Neural networks for node classification on graphs</p>
					</header>
					<p><span style="text-decoration: underline;">T. N. Kipf</span>, M. Welling, <span style="font-weight: bold">Semi-Supervised Classification with Graph Convolutional Networks</span> (ICLR 2017) [<a href='http://arxiv.org/abs/1609.02907'>Link</a>, <a
										href="https://arxiv.org/pdf/1609.02907.pdf">PDF (arXiv)</a>, <a href="https://github.com/tkipf/gcn">code</a>, <a href="http://tkipf.github.io/graph-convolutional-networks/">blog</a>]</p>
				</div>
			</div>
			<hr />
			<div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
					<div class="image rounded"><img src="images/gae.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Variational Graph Auto-Encoders</h4>
						<p>A latent variable model for graph-structured data</p>
					</header>
					<p><span style="text-decoration: underline;">T. N. Kipf</span>, M. Welling, <span style="font-weight: bold">Variational Graph Auto-Encoders</span>, (NeurIPS Bayesian Deep Learning Workshop 2016) [<a href='https://arxiv.org/abs/1611.07308'>Link</a>, <a href="http://arxiv.org/pdf/1611.07308v1.pdf">PDF (arXiv)</a>, <a href="https://github.com/tkipf/gae">code</a>]</p>
				</div>
			</div>
			<hr /> -->
			<!-- <div class="row 200%">
				<div class="3u 4u(large) 12u$(medium)">
					<div class="image rounded"><img src="images/msc.png" width="180" alt="" style="border:none;" /></div>
				</div>
				<div class="9u$ 8u$(large) 12u$(medium)">
					<header>
						<h4>Recurrent Neural Networks for Graph-Based 3D Agglomeration</h4>
						<p>M.Sc. thesis (2016), Department of Connectomics, MPI for Brain Research</p>
					</header>
					<p>Thesis copy available here: [<a href="http://tkipf.github.io/misc/MScThesis_ThomasKipf.pdf">PDF (9 MB)</a>]</p>
				</div>
			</div>
			<hr /> -->
			<p style="text-align:center;">For a full list, have a look at my <a href="https://scholar.google.com/citations?user=5rg6ljIAAAAJ&hl=en">Google Scholar</a> page.</p>
		</div>
	</section>


	<!-- Three -->
	<section id="three" class="wrapper style3 special">
		<div class="container 50%">

			<h2>Curriculum vitae</h2>
			<ul class="actions">
				<li><a href="CV__short_.pdf" class="button alt big">Download CV</a></li>
			</ul>
		</div>
	</section>


	<!-- Footer -->
	<footer id="footer">
		<div class="container">
			<h2>Get in touch</h2>
			<ul class="icons">
				<li>
					<a href="https://scholar.google.com/citations?user=5rg6ljIAAAAJ&hl=en" class="icon fa-graduation-cap"></a>
				</li>
				<li>
					<a href="https://github.com/soroorh" class="icon fa-github"></a>
				</li>
				<li>
					<a href="https://twitter.com/soroorh1" class="icon fa-twitter"></a>
				</li>
				<li>
					<a href="https://de.linkedin.com/in/soroor-hediyeh-zadeh-958086278" class="icon fa-linkedin"></a>
				</li>
			</ul>
			<ul class="copyright">
				<li>&copy; 2023 Soroor Hediyeh-zadeh</li>
				<li>Design: <a href="http://templated.co">TEMPLATED</a></li>
			</ul>
		</div>
	</footer>

</body>

</html>
